modules/op-release-notes-1-15.adoc

```diff
@@ -0,0 +1,451 @@
+
+* Before this update, when using a resolver to incorporate a task from a remote source, the resolver automatically added the `kind` value of `Task` to the resulting  specification. With this update, the resolver does not add a `kind` value to the specification.
+
+* Before this update, when you set configuration options using an `options:` section in the `TektonConfig` CR, these options were sometimes not applied correctly. With this update, the options are applied correctly.
+
+* Before this update, if you set the `enable-api-fields` field and certain other fields in the `TektonConfig` CR, the settings were lost after any update of {pipelines-shortname}. With this update, the settings are preserved during updates.
+
+* Before this update, if you configured the Horizontal Pod Autoscaler (HPA) using the options section in the `TektonConfig` CR, any existing HPA was updated correctly but a new HPA was not created when required. With this update, HPA configuration using the options section works correctly.
+
+* Before this update, you could erroneously change the `targetNamespace` field in the `TektonConfig` CR, creating an unsupported configuration. With this update, you can no longer change this field. Changing the target namespace name from `openshift-pipelines` is not supported.
+
+* Before this update, if the `pipelines-scc-rolebinding` rolebinding was missing or deleted in any namespace, the {pipelines-shortname} Operator controller failed to create default resources in new namespaces correctly. With this update, the controller functions correctly.
+
+* Before this update, when you specified a `namespaceSelector` value when defining a `triggerGroup` in an `EventListener` CR, the event listener was unable to access triggers in the specified namespace if it was not the same as the namespace of the event listener. With this update, the event listener can access triggers in the specified namespace.
+
+* Before this update, when a request was sent to an `EventListener` route URL with a `Content-Type` header, this header was not passed to the interceptor. With this update, the header is passed to the interceptor.
+
+* With this update, several potential causes for {tekton-results} becoming unresponsive, crashing, or consuming a large amount of memory were removed.
+
+* Before this update, in the Pipeline details page of the web console, if a `when` expression using CEL was configured for a task, information was not displayed correctly. With this update, the information is displayed correctly.
+
+* Before this update, in the Pipeline details page of the web console, the menu was not visible when you enabled dark mode in the web console. With this update, the menu is visible.
+
+* Before this update, in the Pipelines page of the web console, information about running statistics of pipelines did not include the information saved in {tekton-results}. With this update, the page includes all available running statistics information for every pipeline.
```

Comment:
Markup the name of the page with asterisks, e.g.:
```
in the *Pipeline* details page
```
Also note the the first two say "Pipeline", but the third one says "Pipelines".

Reply:
marked up, thanks. And the third one is a different page.